src/hevm/src/EVM.hs

```diff
@@ -2333,20 +2327,21 @@ costOfPrecompile (FeeSchedule {..}) precompileAddr input =
     -- MODEXP
     0x5 -> num $ (f (num (max lenm lenb)) * num (max lene' 1)) `div` (num g_quaddivisor)
       where (lenb, lene, lenm) = parseModexpLength input
-            lene' = if lene <= 32 && ez then 0
-                    else if lene <= 32 then num (log2 e')
-                    else if e' == 0 then 8 * (lene - 32)
-                    else num (log2 e') + 8 * (lene - 32)
+            lene'
+              | lene <= 32 && ez = 0
+              | lene <= 32 = num (log2 e')
+              | e' == 0 = 8 * (lene - 32)
+              | otherwise = num (log2 e') + 8 * (lene - 32)
```

Comment:
```suggestion
            lene' | lene <= 32 && ez = 0
                  | lene <= 32 = num (log2 e')
                  | e' == 0 = 8 * (lene - 32)
                  | otherwise = num (log2 e') + 8 * (lene - 32)
```